client/login/wp-login/index.jsx

```diff
@@ -70,10 +70,11 @@ export class Login extends React.Component {
 	}
 
 	render() {
-		const { translate, twoFactorAuthType } = this.props;
+		const { translate, twoFactorAuthType, clientId } = this.props;
+		const title = clientId ? translate( 'Howdy! Log in to ' + clientId + ' with your WordPress.com account.' ) : translate( 'Log in to your account.' );
```

Comment:
Do we really want to display the name of the app at this point? Looking at the current design https://cloudup.com/c41hIMuPnHw it may be a bit confusing. It looks like you are asked to give your WordPress.com credentials to Meteorik. Couldn't we simply have the plain/boring WordPress.com login here? Users will see the name Meteorik when they give their consent on the next screen https://cloudup.com/cwEDZUFLc61